src/hotspot/share/prims/jvmtiEnvBase.cpp

```diff
@@ -1981,7 +1973,7 @@ JvmtiHandshake::execute(JvmtiUnitedHandshakeClosure* hs_cl, jthread target) {
   JavaThread* java_thread = nullptr;
   oop thread_obj = nullptr;
 
-  jvmtiError err = JvmtiEnvBase::get_threadOop_and_JavaThread(tlh.list(), target, &java_thread, &thread_obj);
+  jvmtiError err = JvmtiEnvBase::get_threadOop_and_JavaThread(tlh.list(), target, current, &java_thread, &thread_obj);
```

Comment:
I think a good cleanup would be to also replace `current` with `current_thread`, although I'm not sure how common each are. I see 3 `current` references in this webrev.

Reply:
Looks like in JVMTI `current_thread` is more common (and `current` is usually used in runtime :)

Reply:
The plan is to unify this with the approach used by the Runtime team.

Reply:
Replaced all touched "current_thread" and "calling_thread" with "current"